lambda/package.json

```diff
@@ -10,6 +10,7 @@
     "lodash": "^4.17.5",
     "md5": "^2.2.1",
     "request": "^2.85.0",
+    "require": "^2.4.20",
```

Comment:
Isn't require included by default?

Reply:
idk, it fixed my IDE's complaints

Reply:
Huh, looks to be fixed. I can remove it.

Reply:
Strange..

Reply:
You might have been dealing with the import vs require issue.  We'll discuss it when we meet, because you are probably overriding the node default require module